requirements.txt

```diff
@@ -10,5 +10,4 @@ boto3>=1.4.0
 moto==1.3.7
 Pillow>=3.3.1
 numpy>=1.11.1
-intern>=0.9.7
-
+intern<1.0.0
```

Comment:
if dropping support for py2, i think you could say `intern>=1.0.0` here (<1.0 had py2 support,  ≥1.0 has py3 only)

Reply:
Can't yet. We use the ingest-client on the endpoint and are currently still stuck on 3.5. :(

We could leave at `>=0.9.7` if the boss' requirements.txt says `<1.0.0`.